modules/formulize/class/textElement.php

```diff
@@ -230,6 +231,10 @@ function generateValidationCode($caption, $markupName, $element, $entry_id=false
 		// $entry_id is the ID number of the entry that this data is being saved into. Can be "new", or null in the event of a subformblank entry being saved.
     function prepareDataForSaving($value, $element, $entry_id=null) {
 			$ele_value = $element->getVar('ele_value');
+			// Trim the value if the option is set
+			if ($ele_value[12]) {
```

Comment:
I suggest:
```
if(isset($ele_value[12] AND $ele_value[12]) {
```
The thing with PHP is that referencing a key that does not exist in an array, results in a warning. Didn't used to, but clutters up logs now and theoretically it is a warning because there may be a logic issue in the code if non existent array keys are popping up at runtime. Not in our case, but checking with isset is still worthwhile.